README.rst

```diff
@@ -78,7 +94,7 @@ To run a celery worker:
 
 .. code-block:: bash
 
-    cd sprint_management_application
+    cd sprints
```

Comment:
@Agrendalath, don't we have to run this command the one in the next line inside the container using `docker-compose`?

Reply:
That's correct. I forgot to add it.